### ISSUE TITLE
find bug from my code

### ISSUE THREAD

**Issue by shruthi271096** [shruthi271096/grid-problem-using-bsf-and-python-code#1]
############### Blackjack Project #####################

#Difficulty Normal 😎: Use all Hints below to complete the project.
#Difficulty Hard 🤔: Use only Hints 1, 2, 3 to complete the project.
#Difficulty Extra Hard 😭: Only use Hints 1 & 2 to complete the project.
#Difficulty Expert 🤯: Only use Hint 1 to complete the project.

##################### Hints #####################

#Hint 1: Go to this website and try out the Blackjack game: 
#   https://games.washingtonpost.com/games/blackjack/
#Then try out the completed Blackjack project here: 
#   http://blackjack-final.appbrewery.repl.run

#Hint 2: Read this breakdown of program requirements: 
#   http://listmoz.com/view/6h34DJpvJBFVRlZfJvxF
#Then try to create your own flowchart for the program.

#Hint 3: Download and read this flow chart I've created: 
#   https://drive.google.com/uc?export=download&id=1rDkiHCrhaf9eX7u7yjM1qwSuyEk-rPnt

#Hint 4: Create a deal_card() function that uses the List below to *return* a random card.
#11 is the Ace.
#cards = [11, 2, 3, 4, 5, 6, 7, 8, 9, 10, 10, 10, 10]
from art import logo
from replit import clear
import random
def deal_card():
  cards = [11, 2, 3, 4, 5, 6, 7, 8, 9, 10, 10, 10, 10]
  card=random.choice(cards)
  return card

#Hint 5: Deal the user and computer 2 cards each using deal_card() and append().
#user_cards = []
#computer_cards = []

#Hint 6: Create a function called calculate_score() that takes a List of cards as input 
#and returns the score. 
#Look up the sum() function to help you do this.
def calculate_score(cards):
#Hint 7: Inside calculate_score() check for a blackjack (a hand with only 2 cards: ace + 10) and return 0 instead of the actual score. 0 will represent a blackjack in our game.
  if sum(cards)==21 and len(cards)==2:
    return 0
#Hint 8: Inside calculate_score() check for an 11 (ace). If the score is already over 21, remove the 11 and replace it with a 1. You might need to look up append() and remove().
  if sum(cards)>21 and 11 in cards:
      cards.remove(11)
      cards.append(1)
      return sum(cards)
#Hint 9: Call calculate_score(). If the computer or the user has a blackjack (0) or if the user's score is over 21, then the game ends.
#Hint 10: If the game has not ended, ask the user if they want to draw another card. If yes, then use the deal_card() function to add another card to the user_cards List. If no, then the game has ended.

#Hint 11: The score will need to be rechecked with every new card drawn and the checks in Hint 9 need to be repeated until the game ends.

#Hint 12: Once the user is done, it's time to let the computer play. The computer should keep drawing cards as long as it has a score less than 17.
#Hint 13: Create a function called compare() and pass in the user_score and computer_score. If the computer and user both have the same score, then it's a draw. If the computer has a blackjack (0), then the user loses. If the user has a blackjack (0), then the user wins. If the user_score is over 21, then the user loses. If the computer_score is over 21, then the computer loses. If none of the above, then the player with the highest score wins.
def compare(user_score,computer_score):
  if user_score>21 and computer_score>21:
    return "you went over, you lose"
  elif computer_score==user_score:
     return "it's draw"
  elif computer_score==0:
     "you lose,opponent has ablack jack"
  elif user_score>21:
    return "you went over, you lose"
  elif user_score==0:
    return "you win with black jack"
  elif computer_score>21:
    return "you win,opponen went over"
  elif user_score>computer_score:
    return "you win with highest score"
  else:
    return "you lose,opponent has highest score"
#Hint 14: Ask the user if they want to restart the game. If they answer yes, clear the console and start a new game of blackjack and show the logo from art.py.
def play_game():
  print(logo)
  user_cards=[]
  computer_cards=[]
  is_gameover=False
  for n in range(2):
    user_cards.append(deal_card())
    computer_cards.append(deal_card())
  
  while not is_gameover:
   user_score=calculate_score(user_cards)
   computer_score=calculate_score(computer_cards)
   print(f"your cards{user_cards},current_score:{user_score}")
   print(f"computer's first card:{computer_cards[0]}")
  if user_score==0 or computer_score==0 or user_score>21:
        is_gameover=True
  else:
      user_should_deal=input("you want anothr card drawn if yes press y otherwise pass")
  if user_should_deal=='y':
      user_cards.append(deal_card())
      user_score=calculate_score(user_cards)
  else:
      is_gameover=True
  while computer_score!=0 and computer_score<17:
   computer_cards.append(deal_card())
   computer_score=calculate_score(computer_cards)
  print(f"your final hand is {user_cards} and final score is :{user_score}")
  print(f"computer final hand is {computer_cards} and computer final score is {computer_score}")
  print(compare(user_score,computer_score))
  while input("do you want to play blackjack? if yes type y else exit")=="y":
            clear()
            play_game()
      
#what is error in my code please any one ellaborate
